src/scripts/farming/mutation/mutationTypes/GrowNearFlavorMutation.ts

```diff
@@ -54,23 +54,9 @@ class GrowNearFlavorMutation extends GrowNearMutation {
      * @param idx The plot index
      */
     mutationChance(idx: number): number {
-        const plots = Plot.findNearPlots(idx);
-        if (!plots.length) {
-            return super.mutationChance(idx);
-        }
-        let sameBerries = 0;
-        plots.forEach(idx => {
-            const plot = App.game.farming.plotList[idx];
-            if (!plot.isUnlocked) {
-                return;
-            }
-            if (plot.isEmpty()) {
-                return;
-            }
-            if (plot.berry === this.mutatedBerry) {
-                sameBerries += 1;
-            }
-        });
+        const sameBerries = Plot.findNearPlots(idx).filter(plotIndex => {
+            return App.game.farming.plotList[plotIndex].berry === this.mutatedBerry;
+        }).length;
         return super.mutationChance(idx) * Math.pow(10, -sameBerries);
```

Comment:
Is it expected that this will basically destroy the chances of mutation for the same berry if there is 1 or more?

Reply:
Yea it'll kill the invasiveness. I added 10 just randomly. We could scale it down to 5 or something if you still want some reproduction occurring.

Reply:
I feel like anywhere between `2` and `5` should cut down the odds enough that it doesn't become so invasive,

Reply:
I'll put it at 4 for now, hopefully that'll curb it still.